sdk/storage/storage-queue/package.json

```diff
@@ -118,11 +119,17 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-http": "^3.0.0",
+    "@azure/core-auth": "^1.4.0",
+    "@azure/core-client": "^1.6.2",
+    "@azure/core-http-compat": "^2.0.0",
     "@azure/core-paging": "^1.1.1",
+    "@azure/core-rest-pipeline": "^1.10.1",
     "@azure/core-tracing": "1.0.0-preview.13",
```

Comment:
is core-tracing upgrade happening later?

Reply:
Yeah, first PR is to move off of core-http, second is to move to core-tracing 1.0, and third is to upgrade the test recorder.